ui/partials/perf/graphsctrl.html

```diff
@@ -13,8 +13,8 @@
             </button>
             <table>
               <tr>
-                <td width="10px" class="graph-legend" style="padding: 4px; background-color: {{series.color}}"></td>
-                <td class="graph-legend">
+                <td width=10px  class="graph-legend" style="padding: 4px; background-color: {{series.blockColor}}" ></td>
```

Comment:
I didn't notice this before, but you removed the quotes and added an unwanted space around 10px here. I'll fix this myself, but letting you know for next time. :)